examples/commands-leveldb/src/test/scala/CommandsLevelDB.scala

```diff
@@ -31,9 +31,16 @@ object LevelDBSpec extends Commands {
   }
 
   def canCreateNewSut(newState: State, initSuts: Traversable[State],
-    runningSuts: Traversable[Sut]
-  ) = {
-    !initSuts.exists(_.name == newState.name) &&
+                      runningSuts: Traversable[Sut]
+                     ) = {
+    for {
+      initSut <- initSuts
+      runningSut <- runningSuts
+    } yield (initSut.name == newState.name) || (runningSut.name == newState.name) match {
+      case true => false
+      case false => true
+    }
+
```

Comment:
Isn't this a discarded pure expression?  (I'm surprised scalac didn't complain?  Maybe I'm misreading it?)

Reply:
Can this be merged?

Reply:
I might be missing something, but I still think that for comprehension is just being thrown away.  Does this code work as intended?

Reply:
yeah it works